nautobot/dcim/views.py

```diff
@@ -1309,8 +1310,21 @@ def get_extra_context(self, request, instance):
         }
         RequestConfig(request, paginate).configure(device_table)
 
+        network_driver_names = {
+            "ansible",
+            "hier_config",
+            "napalm",
+            "netmiko",
+            "ntc_templates",
+            "pyats",
+            "pyntc",
+            "scrapli",
+        }
```

Comment:
This probably should be defined somewhere in `nautobot.dcim.constants` so that we don't have to update it in a bunch of different places when a new driver becomes available.

Reply:
I would go one step further and say this should be exposed through `netutils`. I'll go ahead and move to `nautobot.dcim.constants` for now.